config/application.rb

```diff
@@ -1,28 +1,15 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
-require 'active_record/railtie'
-require 'action_controller/railtie'
-require 'action_mailer/railtie'
-require 'rails/test_unit/railtie'
-require 'sprockets/railtie'
+require 'rails/all'
 
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module ReleaseApp
   class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    config.time_zone = 'London'
```

Comment:
I assume this means we're switching to UTC - just wanted to check this is intended as I guess if times from models are displayed they will all change when this is deployed?